_layouts/blog.html

```diff
@@ -12,6 +12,50 @@
       <div class="row pt-4 d-flex align-items-center text-center h-100">
         <div class="col-12 text-white">
           <h1 class="display-3 w-100"> {{ page.title }} </h1>
+          <h4>
+            {% for author in page.authors %}
+              {% if forloop.first %} by {% endif %}
+              {% if author.email %}<a href="mailto:{{ author.email }}">{% endif %}
+              {{ author.name | default: "Anonymous"}}
+              {% if author.email %}</a>{% endif %}
+              {% if author.affiliation %}<small>({{ author.affiliation }})</small>{% endif %}
+
+              <!-- Start Social badges -->
+              {% if author.web %}
+                <a href="{{ author.web }}" target="_blank"><i class="ri-links-fill align-text-bottom"></i></a>
+              {% endif %}
+              {% if author.githubid %}
+                <a href="https://github.com/{{ author.githubid }}" target="_blank"><i class="ri-github-fill align-text-bottom"></i></a>
+              {% endif %}
+
+              {% if author.twitterid %}
+                <a href="https://twitter.com/{{ author.twitterid }}" target="_blank"><i class="ri-twitter-fill align-text-bottom"></i></a>
+              {% endif %}
+
+              {% if author.stackoverflowid %}
+                <a href="https://stackoverflow.com/u/{ author.stackoverflowid }}" target="_blank"><i class="ri-stack-overflow-fill align-text-bottom"></i></a>
```

Comment:
Oops, only just noticed that there is a `{` missing.

Reply:
Will fix

Reply:
See #32